compass-js-frontend/bower.json

```diff
@@ -24,19 +24,29 @@
     "jquery-ui-layout-bower": "~1.4.4",
     "jquery-ui": "~1.11.4",
     "fancytree": "~2.9.0",
-    "fontawesome": "~4.3.0"
+    "fontawesome": "~4.3.0",
+	"basicContext": "~3.0.0"
```

Comment:
whitespace issue. Just as a guess: this was added "by hand" instead of `bower install --save`, right?